src/sfizz/Voice.h

```diff
@@ -494,28 +494,31 @@ class Voice {
 
 inline bool sisterVoices(const Voice* lhs, const Voice* rhs)
 {
-    return lhs->getAge() == rhs->getAge()
-        && lhs->getTriggerNumber() == rhs->getTriggerNumber()
-        && lhs->getTriggerValue() == rhs->getTriggerValue()
-        && lhs->getTriggerType() == rhs->getTriggerType();
+    if (lhs->getAge() != rhs->getAge())
+        return false;
+
+    if (lhs->getTriggerNumber() != rhs->getTriggerNumber())
+        return false;
+
+    if (lhs->getTriggerValue() != rhs->getTriggerValue())
+        return false;
+
+    if (lhs->getTriggerType() != rhs->getTriggerType())
+        return false;
+
+    return true;
 }
 
 inline bool voiceOrdering(const Voice* lhs, const Voice* rhs)
 {
     if (lhs->getAge() > rhs->getAge())
         return true;
-    if (lhs->getAge() < rhs->getAge())
-        return false;
 
```

Comment:
If removing the second check, this makes it incorrect as ordering function. Same of other cases below.

One may also write it:
```
if (a.age != b.age) {
  return a.age > b.age;
}
[...]
return false;
```